doc/index.rst

```diff
@@ -7,6 +7,19 @@ Introduction
 partup is a program used for initializing flash memory on a target device using
 a predefined layout.
 
+Main Features
+-------------
+
+- Partition block devices
+- Automatic partition size calculation to use whole device
+- Supported disklabels: gpt, msdos
+- Supported filesystems: fat32, ext2, ext3, ext4
+- Support for eMMC boot partitions
+- Write files and extract archives to partitions
+- Write raw binaries to device at specified offset
+- Checksum verification for input files (MD5, SHA256)
+- Set PARTUUID and partition flags
```

Comment:
Please align the content of this list to 3 spaces to have the same indentation as other reStructuredText directives:
```rst
-  one
-  two
-  three
```